README.md

```diff
@@ -313,7 +313,18 @@ If you are using the Enterprise Edition of DC/OS with Authentication enabled you
 curl -v -H "Authorization: token=$(dcos config show core.dcos_acs_token)" -X POST http://<dcos_url>/service/cassandra/v1/plan/continue
 ```
 
-## Uninstall
+# Upgrade
+
+1. In the DC/OS web interface, destroy the Cassandra instance to be updated. (This will not kill Cassandra node tasks).
+1. Verify that you no longer see the Cassandra instance in the DC/OS web interface.
+1. From the DC/OS CLI, install the latest version of Cassandra [with any customizations you require](1.7/usage/service-guides/cassandra/install-and-customize/) in a JSON options file:
+    $ dcos package install cassandra --options=/path/to/options.json
+
+   The command above will trigger the install of the new Cassandra version. You can follow the upgrade progress just like the [configuration upgrade progress](/1.7/usage/service-guides/cassandra/configuration/).
```

Comment:
Can you tell me a bit more about the "upgrade process" you refer to up here? Didn't the user just upgrade?